posthog/urls.py

```diff
@@ -168,6 +169,10 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> UR
     opt_slash_path("api/signup", signup.SignupViewset.as_view()),
     opt_slash_path("api/social_signup", signup.SocialSignupViewset.as_view()),
     path("api/signup/<str:invite_id>/", signup.InviteSignupViewset.as_view()),
+    path(
```

Comment:
you shouldn't need to do this, you can register this in the org router here: https://github.com/PostHog/posthog/blob/master/posthog/api/__init__.py#L190

This adds it in the right place, and makes sure `self.organisation_id` exists so you don't have to finagle that